### DISCUSSION TITLE
LOTR Mithril

### DISCUSSION THREAD

**Discussion by Frey426** [Roadhog360/Et-Futurum-Requiem#476] [Improvements]
This mod has some LOTR Mod support by adding raw version of Mithril.  this raw Mithril can be smelted in regular Furnace, but in original LOTR Mod Mithril is only can be smelted in Dwarven forge. I think this thing is breaking original  balance

**Comment by Roadhog360** [comment]
You can use craft tweaker to adjust recipes
